## Problem 1

Twelve LED bulbs are randomly selected from a large shipment, and their lifetimes (in hours) are recorded. The average lifetime in this sample is **1,215 hours**. Assume that the true distribution of bulb lifetimes is normally distributed with known standard deviation $\sigma = 80$ hours.

Test
$$
H_0:\ \mu = 1200 \qquad \text{versus} \qquad H_1:\ \mu \neq 1200
$$

using a significance level of $ \alpha = 0.05 $.

**Questions:**

1. What is the value of the test statistic?
2. What are the critical value(s) for this test?
3. Based on your test, what is your conclusion in the context of the problem?


In [3]:
xbar = 1215
mu = 1200
sigma = 80
alpha = 0.05
n = 12

z = (xbar - mu)/(sigma/sqrt(n))
paste0("z: ", round(z, 3))

crit_val = qnorm(1-alpha/2)
paste0("crit_val: ", crit_val)

if(z > crit_val){
    result = "Reject the null hypothesis"
}else{
    result = "Fail to reject the null hypothesis."
}

paste0("Decision: ", result)


[1] "z: 0.65"

[1] "crit_val: 1.95996398454005"

[1] "Decision: Fail to reject the null hypothesis."

## Problem 2

At a particular movie theater, management believes that **50%** of tickets sold are for action movies, **30%** for comedies, and **20%** for dramas.

A random sample of **400** ticket sales is examined. In this sample, **190** tickets were for action movies, **140** for comedies, and **70** for dramas.

Do these data support the claimed **50–30–20** breakdown of movie types?

Consider testing the relevant hypotheses using a chi-square goodness-of-fit test.
**What is the P-value of your test?**

Give your answer to **2 decimal places**.


In [15]:
obs = c(190, 140, 70)
percentages = c(0.5, 0.3, 0.20)

gof_test = chisq.test(obs, p = percentages)
gof_test


	Chi-squared test for given probabilities

data:  obs
X-squared = 5.0833, df = 2, p-value = 0.07874


## Problem 3

The average time it takes to service a car at a certain oil-change chain is believed to be normally distributed with mean **22.5 minutes**.

A random sample of **18** service times from this chain produced a sample mean of **20.8 minutes** and a sample standard deviation of **3.9 minutes**.

Is there evidence that the true mean service time at this chain is **less than 22.5 minutes**?

Test the relevant hypotheses at the **0.10** level of significance.

Which of the following gives your **test statistic**, **critical value**, and **conclusion**?


In [22]:
mu = 22.5
n = 18
xbar = 20.8
s = 3.9
alpha = 0.10

t_stat = (xbar - mu)/(s/sqrt(n))
paste0("t statistic: ", round(t_stat, 4))

p_value = pt(t_stat, df=n-1)
paste0("P-value: ", round(p_value, 4))

crit_val = qt(alpha, df=n-1)
paste0("Critical t value: ", round(crit_val, 4))

if(t_stat < crit_val){
    result = "Reject the null hypothesis"
}else{
    result = "Fail to reject the null hypothesis"
}
paste0("Decision: ", result)

[1] "t statistic: -1.8494"

[1] "P-value: 0.0409"

[1] "Critical t value: -1.3334"

[1] "Decision: Reject the null hypothesis"

## Problem 4

A random sample of size **250** from **City A** finds **140** adults who regularly use public transportation.

An independent random sample of size **250** from **City B** finds **165** adults who regularly use public transportation.

Consider testing the hypotheses

$$
H_0: p_1 = p_2 \qquad \text{versus} \qquad H_1: p_1 > p_2,
$$

where $p_1$ is the true proportion of adults in City A who regularly use public transportation and $p_2$ is the true proportion in City B.

Using a large-sample z test for the difference in proportions, **find the P-value for this test**.

Give your answer to **3 decimal places**.


In [27]:
p1 = 140
p2 = 165
p1.hat = p1/250
p2.hat = p2/250

p_pool = (p1 + p2) / 500
paste0("p pool: ", round(p_pool, 4))

se = sqrt(p_pool * (1-p_pool) * (1/250 + 1/250))
paste0("se: ", round(se, 4))

z = (p1.hat - p2.hat) / se
paste("z: ", round(z, 4))

# right tailed test
p_value = 1-pnorm(z)
paste0("P-value: ", round(p_value, 4))



[1] "p pool: 0.61"

[1] "se: 0.0436"

[1] "z:  -2.2922"

[1] "P-value: 0.9891"

## Problem 5

The waiting time (in minutes) for customers at a particular coffee shop is believed to be normally distributed with mean 6.2 minutes.

A random sample of **20** customers produced a sample mean waiting time of 5.8 minutes and a sample standard deviation of **2.4** minutes.

Test the hypotheses

$$
H_0:\ \sigma^{2} \le 5 \qquad \text{versus} \qquad H_1:\ \sigma^{2} > 5
$$

at the **0.05** level of significance.

Which of the following is an appropriate rejection rule, and what is your conclusion in the context of the problem?


In [30]:
mu = 6.2
n = 20
xbar = 5.8
s = 2.4
alpha = 0.05

sigma_sq = 5.0
df = n - 1


crit_val = qchisq(1-alpha, df = n-1)
paste0("Critical chi-square value: ", round(crit_val, 4))

S2_crit = sigma_sq * crit_val / df
paste0("Critical S^2 value: ", round(S2_crit, 4))

S2 = s^2
paste0("Sample variance S^2: ", round(S2, 4))

if(S2 > S2_crit){
    result = "Reject the null hypothesis"
}else{
    result = "Fail to reject the null hypothesis"
}
paste0("Decision: ", result)




[1] "Critical chi-square value: 30.1435"

[1] "Critical S^2 value: 7.9325"

[1] "Sample variance S^2: 5.76"

[1] "Decision: Fail to reject the null hypothesis"

## Problem 6

A random sample of size **4** from a $N(\mu_A,\sigma_A^2)$ population has a sample variance of $s_A^2 = 5.4$.

An independent random sample of size **7** from a $N(\mu_B,\sigma_B^2)$ population has a sample variance of $s_B^2 = 10.1$.

Is there evidence to suggest that $\sigma_B^2 > \sigma_A^2$?

Consider testing the hypotheses

$$
H_0:\sigma_B^2 = \sigma_A^2 \quad \text{versus} \quad H_1:\sigma_B^2 > \sigma_A^2
$$

using $\alpha = 0.05$ level of significance and the test statistic

$$
F = \frac{s_B^2}{s_A^2}.
$$

What is the **appropriate critical value** to use for this test?
Give your answer to **3 decimal places**.


In [35]:
round(qf(0.95, df1=6, df2=3), 3)

[1] 8.941

Here’s the solution step-by-step.

---

### Step 1: Set up the test

We’re testing

$$
H_0:\sigma_B^2 = \sigma_A^2 \quad \text{vs} \quad H_1:\sigma_B^2 > \sigma_A^2
$$

using the test statistic

$$
F = \frac{s_B^2}{s_A^2}
$$

Because we put $s_B^2$ on top, the numerator d.f. correspond to sample B.

* Sample A: $n_A = 4 \Rightarrow d.f. = n_A - 1 = 3$
* Sample B: $n_B = 7 \Rightarrow d.f. = n_B - 1 = 6$

So $F \sim F_{6,3}$ under $H_0$.

This is a **right-tailed** test (since $H_1: \sigma_B^2 > \sigma_A^2$).

---

### Step 2: Critical value

For significance level $\alpha = 0.05$, the critical value is

$$
F_{0.95; 6,3}
$$

Using F-distribution tables or software:

$$
F_{0.95; 6,3} \approx 8.941
$$

---

### Step 3: (Optional) Compare with test statistic

$$
F_{\text{obs}} = \frac{s_B^2}{s_A^2}
= \frac{10.1}{5.4} \approx 1.87
$$

Since $1.87 < 8.941$, we would **not** reject $H_0$.

---

**Answer (critical value):**
$$
F_{\text{crit}} \approx \boxed{8.941}
$$
